addons/skin.estuary/xml/Constants_1920.xml

```diff
@@ -7,5 +7,5 @@
 	<constant name="tvrecordings_width">1060</constant>
 	<constant name="eventloglist_width">1430</constant>
 	<constant name="playlisteditorlist_width">770</constant>
```

Comment:
if you intent to push those changes one of these days, i can rebase this pr.

or you could do it the other way around.